CMakeLists.txt

```diff
@@ -104,7 +104,10 @@ if(WIN32)
     add_definitions(-D_WIN32_WINNT=0x0602)
 endif()
 
-if(APPLE)
+if(APPLE AND CMAKE_OSX_ARCHITECTURES MATCHES "arm64")
```

Comment:
Can you call this instead so it'll work if the folder changes?
`brew --prefix`

Reply:
I presume you mean this...?: 
```
if(APPLE)
    include_directories($(brew --prefix)/include)
    link_directories($(brew --prefix)/lib)
endif()
```

That would make much more sense to do, but I can't get it to work. 

Do I have an error with my syntax?

Reply:
Cmake isn't shell script, you'll need to use execute_process https://cmake.org/cmake/help/latest/command/execute_process.html

Reply:
That’s over my head. Someone else would have to implement that.